crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -150,6 +150,13 @@ impl CheckpointExecutor {
                 0
             });
         let mut pending: CheckpointExecutionBuffer = FuturesOrdered::new();
+
+        let mut now_time = Instant::now();
```

Comment:
`epoch_start_time` and `epoch_start_transaction_num`?

Reply:
Epoch start time can be up to a day away, which is way too much smoothing for the purposes of this TPS measurement. I want the "current" TPS, rather than an average since epoch start. As you suggest in the top comment, I could indeed read metrics twice and do a delta over `checkpoint_transaction_count_sum` but again either I need a custom script to do this outside our software, or need to hook up a grafana like DB, which I doubt most operators do.

Reply:
Sounds good. Using a moving window makes sense here. I hope there can be an easy to use local metrics agent + dashboard.